### PR TITLE
Added description, parameters, reference and minor changes

### PR DESCRIPTION
**Pull request by ajinkya1986**

### CODE REVIEW

File: content/docs/06-integrations/00-cloudanix.md [hunk 1]
```diff
@@ -16,27 +16,67 @@ import Tooltip from "shared/components/ui/Tooltip";
...
+This helm chart installs 4 Cloudanix services to enable container security capabilities:
 
-- **inventory-service**
-- **threat-service**
-- **config-cron**
-- **misconfig-cron**
+- **inventory-service** : Inventory service watches for any new Kubernetes resources and displays them in Cloudanix dashboard.
+- **threat-service** : Threat service exports threat events that are visible on the Cloudanix dashboard along with the affected Kubernetes resources.
+- **config-cron** : Config Cron is a job meant to run periodically to maintain the configuration of Cloudanix services (viz. inventory-service and threat-service) running in a Kubernetes cluster.
+- **misconfig-cron** : Captures Kubernetes misconfiguration.
```
Comment (mpurusottamc) [inline content/docs/06-integrations/00-cloudanix.md]: let's add `and displays them in Cloudanix dashboard.`

**Review by mpurusottamc — CHANGES_REQUESTED** [review]
minor changes needed.

File: content/docs/06-integrations/00-cloudanix.md (hunk 1 above)
Comment (ajinkya1986) [reply content/docs/06-integrations/00-cloudanix.md]: Added

**Review by mpurusottamc — APPROVED** [review]
looks good.

**Pull request merged**